lib/template.html

```diff
@@ -5,21 +5,13 @@
 {% endif %}
 
 {% assign seo_site_title = site.title | default: site.name %}
+{% assign seo_page_title = page.title | default: seo_site_title %}
+{% assign seo_title = page.title | default: seo_site_title %}
 
-{% if page.title %}
-  {% assign seo_title = page.title %}
-  {% assign seo_page_title = page.title %}
-
-  {% if seo_site_title %}
-    {% assign seo_title = seo_title | append:" - " | append: seo_site_title %}
-  {% endif %}
-{% elsif seo_site_title %}
-  {% assign seo_title = seo_site_title %}
-  {% assign seo_page_title = seo_site_title %}
-
-  {% if site.description %}
-    {% assign seo_title = seo_title | append:" - " | append: site.description %}
-  {% endif %}
+{% if page.title and seo_site_title %}
+  {% assign seo_title = seo_title | append:" | " | append: seo_site_title %}
+{% elsif site.description and seo_site_title %}
+  {% assign seo_title = seo_title | append:" | " | append: site.description %}
```

Comment:
Can we use `seo_site_title` here, just to make explicit what we are doing?

```
{% assign seo_title = seo_site_title | append:" | " | append: site.description %}
```

We could probably get rid of the assignment on L9 and also change l12 to refer to `page.title`

Since it is what we're testing for, it seems like that should be what we use.